Frontend/ui-customer/src/Components/ChooseYourRide/View.purs

```diff
@@ -45,6 +45,13 @@ view push config =
         , width MATCH_PARENT
         ] <> FontStyle.h1 TypoGraphy)
     , estimatedTimeAndDistanceView push config
+    , textView $
+        [ text $ getString TOLL_CHARGES_WILL_BE_EXTRA
```

Comment:
@Akash2172 can you add a variable which would decide whether to show this text or not instead of passing empty string

Reply:
done